packages/featureflags/featureToggleManager.js

```diff
@@ -0,0 +1,39 @@
+const { GrowthBook } = require('@growthbook/growthbook')
```

Comment:
Not specified in package.json, but used directly

Reply:
done